src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -5,27 +5,16 @@ import { useEthereum } from '@/contexts/EthereumContext';
  * Ethereum-based blockchain. It leverages the useEthereum hook
  * from the EthereumContext for managing blockchain connections.
  */
+
 const ConnectButton: React.FC = () => {
   const { connect, disconnect, account } = useEthereum();
-
   return (
-    <div>
```

Comment:
no need for div -> cf: https://github.com/lukso-network/tools-dapp-boilerplate/pull/19#discussion_r1472515940